hack/release.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# standard bash error handling
+set -o nounset # treat unset variables as an error and exit immediately.
+set -o errexit # exit immediately when a command fails.
+set -E         # needs to be set if we want the ERR trap
```

Comment:
consider also `pipefail`
```
set -o pipefail
```
commands joined by pipe `|` can fail but by default, capturing this is set to false.